src/main/java/net/minecraftforge/client/model/pipeline/VertexLighterSmoothAo.java

```diff
@@ -71,18 +71,36 @@ protected float calcLightmap(float[][][][] light, float x, float y, float z)
         float e1 = 1 + 1e-4f;
         if(ax > 2 - 1e-4f && ay <= e1 && az <= e1)
         {
-            if(x > -2 + 1e-4f) x = -2 + 1e-4f;
-            if(x <  2 - 1e-4f) x =  2 - 1e-4f;
+            if(x < 0)
```

Comment:
How about `x = x < 0 ? -2 + 1e-4f : 2 - 1e-4f;`?

Reply:
Could even go further by doing `x = Math.signum(x) * (2 - 1e4f);`.

Reply:
`x = Math.copySign(2 - 1e4f, x)`

Reply:
Can also just set `ax = 2 - 1e-4f;` here too, then it doesn't need to be done unconditionally afterwards.

Reply:
I did not intend for this PR to be on lighting pipeline optimization. To me at least, further changes to this code are outside scope.

Reply:
Understood